### DISCUSSION TITLE
CC entity Radar 1.20

### DISCUSSION THREAD

**Discussion by Kierbo05** [cc-tweaked/CC-Tweaked#1715] [Q&A] [answered]
Is there any way I can make a sensor or radar or something that detects when a certain mob comes within a certain radius of it? Any addon I can get?

**Comment by piprett** [comment, marked as answer]
For fabric, you can use [Plethora Peripherals](https://modrinth.com/mod/plethora-peripherals). [Ducky Peripherals](https://modrinth.com/mod/ducky-periphs) also has an [entity detector](https://github.com/SamsTheNerd/ducky-periphs/wiki/Entity-Detector), and supports Fabric, Forge and Quilt on 1.20.1.

**Comment by SirEdvin** [comment]
You can use [Universal scanner](https://docs.siredvin.site/UnlimitedPeripheralWorks/peripherals/universal_scanner/) from [UnlimitedPeripheralWorks](https://modrinth.com/mod/unlimitedperipheralworks) mod, that are working on Forge and Fabric
